SimpleClient/Source/Client.h

```diff
@@ -18,304 +19,274 @@ namespace vasily
 class Client : public WinsockInterface
```

Comment:
It should be "Typical implimentation" instead of "Typical realization" I guess...

Reply:
Ok, but may be there are some code issues?)